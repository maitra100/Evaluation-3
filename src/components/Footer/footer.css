#footer{
    color: white;
    padding-left: 150px;
    padding-right: 150px;
    display: flex;
    align-items: center;
    height: 60px;
    font-size: 18px;
    justify-content: space-between;
}

#button1{
    width: 25px;
    height: 25px;
    border-radius: 15%;
    background-color: purple;
    margin-left: 10px;
    border: 1px solid purple;
}

#inside{
    display:flex;
    align-items: center;
}

#button2{
    width:25px;
    height:25px;
    border-radius: 15%;
    background-color: blue;
    margin-left:10px;
    border: 1px solid blue;
}

#button3{
    width:25px;
    height:25px;
    border-radius: 15%;
    background-color: grey;
    margin-left:10px;
    border: 1px solid grey;
}

@media only screen and (max-width:580px){
    #footer{
        padding-left: 35px;
    padding-right: 50px;
    }
}




