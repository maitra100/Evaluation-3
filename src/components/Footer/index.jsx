import React from 'react';
import './footer.css';

function Footer() {
  return (
    <header>
      <div id="footer">
        <div id="inside">
          <p>THEMES</p>
          <button id="button1"></button>
          <button id="button2"></button>
          <button id="button3"></button>
        </div>
      </div>
    </header>
  );
}

export default Footer;
