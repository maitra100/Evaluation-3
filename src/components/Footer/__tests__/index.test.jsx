import { render } from '@testing-library/react';
import React from 'react';
import Footer from '../../Footer';

describe('Header', () => {
  it('renders correctly', () => {
    const screen = render(<Footer />);
    expect(screen.asFragment()).toMatchSnapshot();
  });
});
