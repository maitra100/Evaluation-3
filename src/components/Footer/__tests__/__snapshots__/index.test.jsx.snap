// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<DocumentFragment>
  <header>
    <div
      id="footer"
    >
      <div
        id="inside"
      >
        <p>
          THEMES
        </p>
        <button
          id="button1"
        />
        <button
          id="button2"
        />
        <button
          id="button3"
        />
      </div>
    </div>
  </header>
</DocumentFragment>
`;
